dev-tools/mage/kubernetes/kuberemote.go

```diff
@@ -180,24 +183,30 @@ func (r *KubeRemote) syncSSHKey() error {
 
 // deleteServiceAccount syncs required service account.
 func (r *KubeRemote) deleteServiceAccount() {
-	_ = r.cs.RbacV1().ClusterRoleBindings().Delete(r.name, &metav1.DeleteOptions{})
-	_ = r.cs.RbacV1().ClusterRoles().Delete(r.name, &metav1.DeleteOptions{})
-	_ = r.cs.CoreV1().ServiceAccounts(r.namespace).Delete(r.svcAccName, &metav1.DeleteOptions{})
+	_ = r.cs.RbacV1().ClusterRoleBindings().Delete(context.Background(), r.name, metav1.DeleteOptions{})
+	_ = r.cs.RbacV1().ClusterRoles().Delete(context.Background(), r.name, metav1.DeleteOptions{})
+	_ = r.cs.CoreV1().ServiceAccounts(r.namespace).Delete(context.Background(), r.svcAccName, metav1.DeleteOptions{})
```

Comment:
Nit. In functions when multiple calls require a context, define a single context for them, this way it will be easier to change later if/when a context is provided.
```suggestion
      ctx := context.TODO()
	_ = r.cs.RbacV1().ClusterRoleBindings().Delete(ctx, r.name, metav1.DeleteOptions{})
	_ = r.cs.RbacV1().ClusterRoles().Delete(ctx, r.name, metav1.DeleteOptions{})
	_ = r.cs.CoreV1().ServiceAccounts(r.namespace).Delete(ctx, r.svcAccName, metav1.DeleteOptions{})
```